modules/social_features/social_album/social_album.info.yml

```diff
@@ -0,0 +1,10 @@
+name: 'Social Album'
+description: 'Provides Album content type and related configuration.'
+type: module
+core: 8.x
+package: Social
```

Comment:
@ronaldtebrake wondering if ew should add it as Experimental first?

Reply:
I've marked it as experimental.